src/patternfly/components/List/list.scss

```diff
@@ -43,4 +83,16 @@
       }
     }
   }
+
+  &.pf-m-bordered {
+    > * {
+      padding-bottom: var(--pf-c-list--m-bordered--li--PaddingBottom);
+      border-bottom: var(--pf-c-list--m-bordered--li--BorderBottomWidth) solid var(--pf-c-list--m-bordered--li--BorderBottomColor);
+    }
+
+    > :last-child {
+      padding-bottom: 0;
```

Comment:
This is fine, but typically we try to redefine properties via custom properties. So:

```
--pf-c-list--m-bordered--li--PaddingBottom: 0;
--pf-c-list--m-bordered--li--BorderBottomWidth: 0;
```